doc/integrator/print.rst

```diff
@@ -0,0 +1,217 @@
+.. _integrator_print:
+
+Print
+=====
+
+Adding the *print* feature to a c2cgeoportal project involves adding
+a ``cgxp.plugins.Print`` plugin to the viewer, and configuring the MapFish
+Print.
+
+The Print plugin
+----------------
+
+The viewer should include a ``Print`` plugin for the *print* feature to
+be available in the user interface.
+
+See the `Print API doc
+<http://docs.camptocamp.net/cgxp/2.3/lib/plugins/Print.html>`_ for the
+list of options the plugin can receive.
+
+*The main viewer includes a Print plugin by default.*
+
+Using MapFish Print v3
+----------------------
+
+Migration from the v2
```

Comment:
This should be done relay before the 2.3 upgrade ...

Reply:
OK, I will add a line to requirements.rst for this